daemon/src/taker_cfd.rs

```diff
@@ -162,14 +179,13 @@ where
             timestamp: Timestamp::now(),
         };
 
-        self.current_pending_proposals.insert(
-            proposal.order_id,
-            UpdateCfdProposal::RollOverProposal {
-                proposal: proposal.clone(),
-                direction: SettlementKind::Outgoing,
-            },
-        );
-        self.send_pending_update_proposals().await?;
+        let new_proposal = UpdateCfdProposal::RollOverProposal {
+            proposal: proposal.clone(),
+            direction: SettlementKind::Outgoing,
+        };
+        self.projection_actor
```

Comment:
Are we on purpose on inserting here?

Reply:
oops, I missed that one! thanks for spotting! added the missing `insert`